kafkaApplicationSimulator/controllers/kafkaController.js

```diff
@@ -1,4 +1,5 @@
 const { Kafka } = require('kafkajs');
+const { calculateThroughput } = require('kafkaq');
```

Comment:
where is this coming from?

Reply:
my bad, that was a fake thing i was using to explain to nicole. i will delete this

Reply:
done